content/applications/general/calendars/google/google_calendar_credentials.rst

```diff
@@ -2,89 +2,130 @@
 Synchronize Google Calendar with Odoo
 =====================================
 
-Odoo is perfectly integrated with Google Calendar so that you 
-can see & manage your meetings from both platforms 
-(updates go through both directions).
+Synchronize Google Calendar with Odoo to see and manage meetings from both platforms (updates go
+in both directions). This integration helps organize your schedule so you never miss a meeting.
 
 Setup in Google
 ===============
-- Go to `Google APIs platform <https://console.developers.google.com>`__ 
-  to generate Google Calendar API credentials. Log in with your Google account. 
 
-- Go to the API & Services page.
+Enable Google Calendar API
+--------------------------
 
-.. image:: media/google_calendar_credentials00.png
-    :align: center
+To begin, you need to create a new Google API project and enable the Google Calendar API. First, go
+to the `Google API Console <https://console.developers.google.com_>`_ and log into your Google
+account.
```

Comment:
• Remove accusatory 2nd-person (unnecessary)
• "To begin..." --> "First..." is redundant

```suggestion
Begin by creating a new Google API project and enabling the Google Calendar API. Go
to the `Google API Console <https://console.developers.google.com_>`_ and log into your Google
account.
```